docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -388,7 +388,7 @@ The following table shows a comparison of capabilities supported by {beats} and
 |{y}
 |{n}
 |{y}
-|The Integrations app requires a network connection to the {fleet-guide}/fleet-overview.html#package-registry-intro[Elastic Package Registry]. We are considering an https://github.com/elastic/integrations/issues/1178[on-prem version of EPR]. {fleet}-managed {agent}s require a connection to our artifact repository for agent binary upgrades. These are not required for standalone {agent}s or {beats}.
+|The Integrations app requires a network connection to the {fleet-guide}/fleet-overview.html#package-registry-intro[{package-registry}], or a {fleet-guide}/air-gapped.html#air-gapped-diy-epr[self-managed deployment of the {package-registry}]. {fleet}-managed {agent}s require a connection to our artifact repository for agent binary upgrades. These are not required for standalone {agent}s or {beats}.
```

Comment:
These can be local links now, right?

Reply:
Umm, not sure, the document is in the fleet guide. There are other links in this file to the fleet guide using this kind of non-local links.

Reply:
Hmm....since CI is passing, just go ahead and leave them as they are and if changes are needed later, I'll make them.